api_extension(update)

```diff
@@ -0,0 +1,15 @@
+from flask_restx import Api
+api = Api(	
+    title="Mentorship System API",	    
+    version="1.0",	    
+    description="API documentation for the backend of Mentorship System. \n \n"	   
+    + "Mentorship System is an application that matches women in tech to mentor each other, on career development, "	   
+    + "through 1:1 relations during a certain period of time. \n \n"	   
+    + "The main repository of the Backend System can be found here: https://github.com/anitab-org/mentorship-backend \n \n"	   
+    + "The Android client for the Mentorship System can be found here: https://github.com/anitab-org/mentorship-android \n \n"
+    + "For more information about the project here's a link to our wiki guide: https://github.com/anitab-org/mentorship-backend/wiki"	   
+    + "For flutter client for the Mentorship System can be found here: https://github.com/anitab-org/mentorship-flutter"
+    + "For ios client for the Mentorship System can be found here: https://github.com/anitab-org/mentorship-ios"
```

Comment:
These 2 lines are correct :) but the file you put them on is not. You are creating a new file, when what its desired is to edit a specific file called `/app/api/api_extension.py` and this is where you have to add those 2 lines.

Line: https://github.com/anitab-org/mentorship-backend/blob/develop/app/api/api_extension.py#L11

To test this change you have to run the project, so please follow the instructions on the README to do so, and then provide a screenshot of your successful change showing it :)